### ISSUE TITLE
Allow disabling rules at block level

### ISSUE THREAD

**Issue by btilford** [CSSLint/csslint#595]
It would be nice to be able to disable rules per block (or line). Currently a lot of false positives are being generated due to limitiations or bugs in csslint like reporting no fallback bg color for 
<pre><code>
background-color: transparent;
background-color: rgba(0, 0, 0, 0.5);
</code></pre>

Due to the complexity of some of the rules it's quite probable they will never get fixed or new ones will crop up. Being able to disable these rules when a false positive is encountered would be really useful.

Currently were able to use other code style and linters to reject a pull request but with css we are unable to because there are to many false positives reported by csslint.

**Comment by replete** [comment]
I've spent around 3 hours with CSSlint trying to make it viable for production use.

The one thing that would make it more viable is the ability to over-ride CSSlint mistakes. 

If I could just add a simple `/*csslint ignoreline */` or something like
```css
/*csslint ignore:true */
@-ms-viewport { width: device-width; } 
@viewport { width: device-width; } 
/*csslint ignore:false */
```

I would be _so_ happy. I would feel like csslint wasn't a waste of time.

**Comment by a-x-** [comment]
:+1:

**Comment by a-x-** [comment]
+ #607

**Comment by XOP** [comment]
:+1:

**Comment by Sciscitator** [comment]
+1

**Comment by XOP** [comment]
After some thinking I've decided to drop CSS Lint in favor of far way flexible Stylelint (https://github.com/stylelint/stylelint) built on top of PostCSS.

**Comment by diogosg** [comment]
+1

**Comment by frvge** [comment]
Fixed in #630 .
